.gitignore

```diff
@@ -78,3 +78,5 @@ forge-gui/tools/AllCards.json
 forge-gui/tools/EditionTrackingResults
 forge-gui/tools/PerSetTrackingResults
 
+forge-gui/res/cardsfolder/upcoming/herald_of_vengeance.txt
```

Comment:
your IDE probably added those lines

Reply:
Yeah, these cards are not finished yet, that's why I deleted it

Reply:
Just gonna keep this as reminder open
You can use it locally like that, but it shouldn‘t be made part of the master repo or it might cause problems down the line